### PR TITLE
Clarify snapping behaviour in Dodge the Creeps tutorial

### PR DESCRIPTION
**Pull request by kjoshi**
When following the "Dodge the Creeps" tutorial it was not obvious that the "Use Snap" button also needed to be selected to get snapping to work.
(As discussed in: https://github.com/godotengine/godot/issues/15846)

This PR aims to clarify how snapping works for new users following the tutorial

### CODE REVIEW

**Comment by akien-mga** [comment]
Sorry for the late review.

Changes seem fine, but I think it's getting a bit too descriptive, maybe a screenshot would be good instead of a graphical description of the icons?

**Comment by ProHiScore** [comment]
Hello just ran into this problem where I did not even know there was a Toggle snap button, all you need is a part where it explains there is a button there or hotkey.

("make sure you Toggle Snapping with the button to the left of the three vertical dots or with (Shift+s) and make sure “Snap to Grid” is checked . . .")

File: getting_started/step_by_step/your_first_game.rst [hunk 1]
```diff
@@ -667,10 +667,12 @@ you will see some new buttons at the top of the editor:
...
-the points at the corners shown. To have the points snap to the grid, make sure "Snap to
-Grid" is checked. This option can be found under the "Snapping options"
-button to the left of the "Lock" button, appearing as a series of three
-vertical dots.
+the points at the corners shown. To have the points snap to the grid, first make
+sure that "Use Snap" is turned on, and then check that "Snap to Grid" is checked in
+the "Snapping options". The "Use Snap" button can be found to the right
+of the "Pan Mode" button, and has a small magnet symbol with three small dots. 
+The "Snapping options" button is immediately to the right of the "Use Snap" button,
+and appears as a series of three vertical dots.
```
Comment (mhilbrunner) [inline getting_started/step_by_step/your_first_game.rst:675]:
```suggestion
the points at the corners shown. To have the points snap to the grid, first make
sure that "Use Snap" is turned on, and check that "Snap to Grid" is checked in
the "Snapping options".
```

**Comment by NathanLovato** [comment]
I took the liberty of addressing the merge conflicts and updating the wording. The picture still only highlights one option, so it'd be best to update it.

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**